### PR TITLE
Some fixes in FastBanCommand.kt and update the request URL on MalUtils.kt

### PR DESCRIPTION
**Pull request by hechfx**

### CODE REVIEW

File: loritta-plugins/mal-commands/src/main/kotlin/net/perfectdreams/loritta/plugin/malcommands/util/MalUtils.kt [hunk 1]
```diff
@@ -31,14 +31,14 @@ object MalUtils {
     }
 
     private fun queryAnime(q: String): String? {
-        val document = requestDom("search/all?q=${q.encodeToUrl()}")
+        val document = requestDom("anime.php?q=${q.encodeToUrl()}")
         try {
             // The first article would be the anime queries
```
Comment (Nightdavisao) [inline loritta-plugins/mal-commands/src/main/kotlin/net/perfectdreams/loritta/plugin/malcommands/util/MalUtils.kt:36]: Change those comments regarding scrapping of MAL's search page as they are outdated now

File: loritta-plugins/mal-commands/src/main/kotlin/net/perfectdreams/loritta/plugin/malcommands/util/MalUtils.kt [hunk 2]
```diff
@@ -31,14 +31,14 @@ object MalUtils {
...
             // For now, we only need the first anime found from queries
-            val animeArticle = document!!.select("article > .list > .information > a").first()
+            val animeArticle = document!!.select("table[cellpadding=\"0\"][cellspacing=\"0\"] > tbody > tr > td > a[href][class*=\"hoverinfo_trigger\"]").first()
```
Comment (Nightdavisao) [inline loritta-plugins/mal-commands/src/main/kotlin/net/perfectdreams/loritta/plugin/malcommands/util/MalUtils.kt:38]: Rename this variable to something describing what this is exactly being extracted, like "firstAnime"

File: loritta-plugins/mal-commands/src/main/kotlin/net/perfectdreams/loritta/plugin/malcommands/util/MalUtils.kt [hunk 3]
```diff
@@ -31,14 +31,14 @@ object MalUtils {
...
             // Now we just need to get the "a" element and then, get the "href" attribute
             if (animeArticle != null) logger.debug { "Got the element \"a\"!" }
-            val result = animeArticle!!.attr("href")
+            val result = animeArticle!!.attributes()["href"]
```
Comment (Nightdavisao) [inline loritta-plugins/mal-commands/src/main/kotlin/net/perfectdreams/loritta/plugin/malcommands/util/MalUtils.kt:41]: I think you could leave this as before

**Comment by xGabrielDEV** [comment]
I think you could add a more descriptive title to this PR, @Nightdavisao already reviewed it.

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/FastBanCommand.kt [hunk 4]
```diff
@@ -34,42 +37,48 @@ object FastBanCommand {
...
-                                "Cadê a menção fera? ;)"
-                        )
-                )
             } else {
-                if (reason == null) {
+                if (userToBePunished == null) {
                     reply(
                             LorittaReply(
-                                    "Cadê o motivo fera? ;)"
+                                    "Cadê a menção fera? ;)"
```
Comment (xGabrielDEV) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/FastBanCommand.kt:44]: You could fail the command here and prevent it from continuing.

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/FastBanCommand.kt [hunk 5]
```diff
@@ -34,42 +37,48 @@ object FastBanCommand {
                                 "Você não pode usar o meu super comandinho de banir as pessoas com motivos bonitinhos ;w;"
```
Comment (xGabrielDEV) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/FastBanCommand.kt:37]: You could fail the command here and prevent it from continuing.

**Pull request merged**